src/Gui/Tree.cpp

```diff
@@ -2833,6 +2833,7 @@ TreePanel::TreePanel(const char *name, QWidget* parent)
             this, SLOT(showEditor()));
 
     this->searchBox = new Gui::ExpressionLineEdit(this,true);
+    static_cast<ExpressionLineEdit*>(this->searchBox)->setMatchExact(false);

```

Comment:
This line doesn't make sense to me as it actively disallows to customize the search behaviour by setting the parameter value CompleterMatchExact